### ISSUE TITLE
Question about voxel size

### ISSUE THREAD

**Issue by pycoco** [Pointcept/PointTransformerV3#51]
Is it supported different voxel size  for x,y,z in PTv3?

**Comment by pycoco** [comment]
> Yes, of course. Adjust grid size in both data transforms and model configs.

thanks! And if i use zyx order for grid_coords, where do I need to change the code in the ptv3 pipeline?  My training loss does not converge， could u give me some suggestions?

**Comment by Gofinge** [comment]
> And if i use zyx order for grid_coords, where do I need to change the code in the ptv3 pipeline

Just modified the serialization encoding code as "z-trans" or "hilbert-trans".

> My training loss does not converge

There is not too much difference among orders with different axis priorities. The main purpose of introducing difference orders is to enable patch interactive like shift-window.

**Comment by pycoco** [comment]
thanks again, actually i try to modify "z-trans" or "hilbert-trans" related code, but the loss problem did not be fixed. Maybe I need to double check。and really look forward to you releasing the detection-related code and config as soon as possible.

**Comment by Gofinge** [comment]
Noted. BTW: code and config for detection is almost aligned with FlatFormer and is not build upon Pointcept (It will take a lot of time to make Pointcept to fully support object detection).

**Comment by pycoco** [comment]
thanks!  i will have a try.
